fake_switches/juniper_mx/juniper_mx_netconf_datastore.py

```diff
@@ -68,35 +68,112 @@ def member_list_trunk_vlan_error(self, port):
     def validate_vlan_config(self, port, vlan_list):
         pass
 
-    def parse_interfaces(self, conf, etree_conf):
-        handled_elements = []
-        for interface_node in etree_conf.xpath("interfaces/interface/name/.."):
-            handled_elements.append(interface_node)
+    def parse_interface(self, conf, interface_node):
+        port_name = val(interface_node, "name")
 
-            port_name = val(interface_node, "name")
-
-            port = conf.get_port_by_partial_name(port_name)
-            if port is None and re.match("^ae\d+$",port_name):
-                port = self.original_configuration.new("AggregatedPort", port_name)
-                port.vendor_specific["has-ethernet-switching"] = True
+        if port_name == "irb":
```

Comment:
you added a constant for "irb" in the datastore for the juniper qfx, maybe use one here too?

Reply:
it's not a constant, it's a list of protected interfaces